sources/Outputs/include/Job.h

```diff
@@ -104,8 +106,8 @@ class Job {
   * @param networkFileEntry path to the input network file
   * @param outputDir the output directory
   */
-  static void exportJob(const boost::shared_ptr<job::JobEntry>& jobEntry, const boost::filesystem::path& networkFileEntry,
```

Comment:
why change the signature of the method ?

Reply:
Because I had to access a member attribute inside this method to ensure that we would output the new precision only if it was set by the user. Otherwise Dynaflow would use the same value as Dynawo and there would be no need to write it again

Reply:
Please pass this information as a parameter of the method

Reply:
It's back to static